Week-5/Exercise5_2.js

```diff
@@ -0,0 +1,13 @@
+function vowelCount(str){
+    const vowelsMap = new Map();
```

Comment:
Instead of empty map, can create an initial map with a,e,i,o,u and initialise value to 0.
Reason being that for is:

1. Empty String
2. String with missing some vowels.